fabric-api-base/src/main/java/net/fabricmc/fabric/api/util/TriState.java

```diff
@@ -136,4 +143,18 @@ public <X extends Throwable> boolean orElseThrow(Supplier<X> exceptionSupplier)
 
 		throw exceptionSupplier.get();
 	}
+
+	/**
+	 * Value of this enum as string.
+	 *
+	 * @return lowercase name of the value.
+	 */
+	@Override
+	public String asString() {
+		return this.name;
+	}
+
+	TriState(String name) {
```

Comment:
Nit: I would put the constructor after the fields and before the methods.

Reply:
Tbf I feel that the field could be moved down there just to be consistent. While this isn't really java-style, the constructor / private fields are purely implementation details that are useless to pretty much all mod devs (the enum values and all extra methods are only ones important).

Reply:
On the other hand, most Java programmers expect a class' members to be in a certain order, at least for me, changing the order makes me a do a double-take when reading it